tasks/apply_ufw_rules.yml

```diff
@@ -21,3 +21,4 @@
     from: "{{ from_server_ip.msg | default(item.from) }}"
     to: "{{ item.to | default('any') }}"
     comment: "{{ item.rule }} {{ env }} {{ project }} {{ type }} {{ item.comment }} from {{ item.from }} ip on {{ item.proto }} protocol"
+    delete: "{{ delete_ufw_rules | default(false) }}"
```

Comment:
Instead of defining one variable delete_ufw_rules, it would be better to change the ufw rule to add delete: true, this change will also let us disable any ufw port specifically